src/test/java/IMDbBirthdays.java

```diff
@@ -0,0 +1,133 @@
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.WebDriverRunner;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class IMDbBirthdays {
+
+    @BeforeClass
+    public void setUp() {
+
+        Configuration.browser = "chrome";
+        Configuration.browserSize = "1920x1080";
+    }
+
+    @Test
+    public void fetchCelebritiesWithTodayBirthday() {
+        openIMDbHomepage();
+        openBornTodayMenu();
+        ElementsCollection celebrities = fetchCelebrityElements();
+        printCelebrityInfo(celebrities);
+    }
+
+    private void openIMDbHomepage() {
+        Selenide.open("https://www.imdb.com");
+        System.out.println("IMDb homepage opened.");
+    }
+
+
+    private void openBornTodayMenu() {
+        clickMenuButton();
+        clickBornTodayLink();
+    }
+
+
+    private void clickMenuButton() {
+        try {
+            WebElement menuButton = Selenide.$(By.id("imdbHeader-navDrawerOpen"));
```

Comment:
Instead you could've used `SelenideElement menuButton = $("#imdbHeader-navDrawerOpen")`